cypress/integration/version/restart_modal.ts

```diff
@@ -36,7 +53,7 @@ describe("Restarting a version", () => {
 
   it("Selecting on the patch status filter should toggle the tasks that have matching statuses to it", () => {
     cy.get(statusFilter).click();
-    cy.getInputByLabel("All").click({ force: true });
+    cy.get(".cy-checkbox").contains("All").as("target").click({ force: true });
```

Comment:
Why did we make this change?

Reply:
I don't remember honestly, I did it a while back. Is it concerning?

Reply:
I think it no longer worked given the new setup

Reply:
fixed